tkupchyn/homework_08/warior.py

```diff
@@ -0,0 +1,39 @@
+# Details:  https://py.checkio.org/mission/the-warriors/share/e171c1a99e4e63b4551cfd820252edcd/
+
+
+class Warrior:
+
+    health = 50
+    attack_damage = 5
+
+    @property
+    def is_alive(self) -> bool:
+        return self.health > 0
+
+    def attack(self, enemy):
+        enemy.health -= self.attack_damage
+
+
+class Knight(Warrior):
+    def __init__(self):
+        super().__init__()
+        self.attack_damage = 7
+
+
+def fight(unit_1: Warrior, unit_2: Warrior) -> bool:
+    """
+    Returns the result of fight between two units. True if the first unit wins, else False
+    """
+
+    fight_result = False
```

Comment:
you do not need to track fight result, since fight result is weather 1 warrior is alive or not